Podfile

```diff
@@ -51,7 +51,7 @@ end
 
 def wordpress_kit
   # pod 'WordPressKit', '~> 13.0'
-  pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', commit: 'dfb2134ca3b91233d44b3551d91844bf2087996f'
+  pod 'WordPressKit', git: 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', commit: '3df47e113c7647b76c049180e39489e3af7a27dc'
```

Comment:
This commit points to https://github.com/wordpress-mobile/WordPressKit-iOS/pull/727 which, I'm happy to say, seems to address the reader issue that the UI tests picked up in previous runs (see #22612)